cpp/libs/gst_nmos_sender_plugin/utils.cpp

```diff
@@ -0,0 +1,192 @@
+#include "utils.h"
+#include "ossrf/nmos/api/nmos_client.h"
+#include "bisect/nmoscpp/configuration.h"
+#include "bisect/expected/macros.h"
+#include "bisect/expected.h"
+#include "bisect/json.h"
+#include <memory>
+
+using namespace bisect;
+using json = nlohmann::json;
+
+ConfigFields create_default_config_fields()
```

Comment:
We've been using the following conventions:

types: `this_is_a_type_t`, in snake-case and with a `_t` suffix.
member variables (of a class): `this_is_a_member_variable_`, with a `_` suffix.
member variables (of a struct, when it's just a "bag-of-data"): `this_is_a_member_variable`, with no suffix.